.github/workflows/check-tests.yml

```diff
@@ -0,0 +1,14 @@
+name: Pre-Commit Hook
+
+on:
+  pull_request:
+    paths:
+      - 'apps/**'
+      - 'libs/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: ../../scripts/check-test.sh
```

Comment:
Can we somehow pipe to here only the modified files in the PR? Wonder how much time it will take to analyze the whole repo